components/src/components/SimpleDateTime/Component.ts

```diff
@@ -1,4 +1,5 @@
 /* tslint:disable */
+import _ from 'lodash';
```

Comment:
Is this a necessary import? I don't see this being used in any of the changed lines.

Reply:
Forgot to remove from tests. Added new commit removing it.